src/NuGetGallery.Services/UserManagement/UserService.cs

```diff
@@ -420,14 +420,14 @@ public async Task CancelChangeEmailAddress(User user)
             await UserRepository.CommitChangesAsync();
         }
 
-        public virtual async Task ChangeMultiFactorAuthentication(User user, bool enableMultiFactor)
+        public virtual async Task ChangeMultiFactorAuthentication(User user, bool enableMultiFactor, string referrer)
```

Comment:
Is the difference between the interface and this method intentional? If not, consider:

```suggestion
        public virtual async Task ChangeMultiFactorAuthentication(User user, bool enableMultiFactor, string referrer = null)
```